components/network/ntp/Makefile

```diff
@@ -44,6 +45,7 @@ COMPONENT_FMRI=	service/network/ntp
 COMPONENT_CLASSIFICATION=	System/Services
 COMPONENT_LICENSE=	ntp license
 COMPONENT_LICENSE_FILE=	COPYRIGHT
+COMPONENT_BUILD_ARGS=	-j20
```

Comment:
Please _never_ do this.  Use `USE_PARALLEL_BUILD` instead.

Reply:
removed